ext/hal/altera/altera_hal/HAL/inc/altera_common.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
```

Comment:
Why are we doing full license boilerplate here, instead of doing SPDX-License-Identifier?

Reply:
this is not an original Zephyr file, we need to keep the original license boilerplate.

Reply:
anas, altera_common.h is added by me for Zephyr adoption of Altera HAL.

But do we need to keep SPDX or Intel license? I thought it's in external folder and thought Intel license header would be good.

Reply:
Andrew, the new ones are working fine. I can remove the old ones in reworked patches.